### PR TITLE
Update Documentation for Loading Custom Plugins

### PR DESCRIPTION
**Pull request by dyastremsky**
This ticket will update documentation once https://github.com/triton-inference-server/tensorrt_backend/pull/25 is merged.

### CODE REVIEW

**Comment by dyastremsky** [comment]
@tanmayv25 Would "+" be a good delimiter for users to load multiple custom plugins via the backend config?

**Comment by CoderHam** [comment]
I'd recommend ";"

**Comment by dyastremsky** [comment]
Thanks! Easy enough change. :)

File: README.md [hunk 1]
```diff
@@ -50,13 +50,16 @@ main Triton [issues page](https://github.com/triton-inference-server/server/issu
 The command-line options configure properties of the TensorRT
 backend that are then applied to all models that use the backend.
 
-##### --backend-config=tensorrt,coalesce-request-input=\<boolean\>
+##### --backend-config=tensorrt,coalesce-request-input=\<boolean\>,plugins=/path/plugin1.so;/path2/plugin2.so
 
-Instruct TensorRT to consider the requests' inputs with the same name as
+Coalesce-request-input instructs TensorRT to consider the requests' inputs with the same name as
```
Comment (tanmayv25) [inline README.md:55]: Make it "coalesce-request-input" with a lower case c.

**Comment by tanmayv25** [comment]
Looks good to me. ";" is usually used for delimiting the paths.

File: README.md (hunk 1 above)
Comment (dyastremsky) [reply README.md]: Thanks for the feedback. Done.

**Comment by dyastremsky** [comment]
We can't use a semicolon, because the command line will read it as a separate command. The user would then have to use an escape character beforehand, but it seems like our parser turns that into \\;

I think this is why I ended up using a plus sign. Through testing, it looks like a semicolon will work with single quotes (-backend-config='/path/to/plugin;/path2/to/plugin2'), but that deviates from our typical backend-config command behavior. Would you prefer the single quotes or a different delimiter? @tanmayv25 @CoderHam

File: README.md [hunk 2]
```diff
@@ -50,13 +50,16 @@ main Triton [issues page](https://github.com/triton-inference-server/server/issu
 The command-line options configure properties of the TensorRT
 backend that are then applied to all models that use the backend.
 
-##### --backend-config=tensorrt,coalesce-request-input=\<boolean\>
+##### --backend-config=tensorrt,coalesce-request-input=\<boolean\>,plugins=/path/plugin1.so;/path2/plugin2.so
```
Comment (CoderHam) [inline README.md:53]: Use quotes around plugin paths if needed as per your earlier comments

**Review by CoderHam — COMMENTED** [review]
`;` is the native way to separate paths on windows.
`:` is the native way to separate paths on linux and the other candidate to consider. 
I will let you choose which of these you think is better.

I would rather use one of these and ask users to use quotes if needed than use another path separator. 
PS `;` separated list of paths is same way paths are passed in [cmake](https://cmake.org/cmake/help/latest/command/cmake_path.html).

**Review by tanmayv25 — APPROVED** [review]

**Pull request merged**